### ISSUE TITLE
Merge AzCopy proxy settings with Storage Explorer proxy settings

### ISSUE THREAD

**Issue by purpler3m** [microsoft/AzureStorageExplorer#8089]
### Preflight Checklist

- [X] I have installed the [latest version of Storage Explorer](https://github.com/Microsoft/AzureStorageExplorer/releases/latest).
- [X] I have checked existing resources, including the [troubleshooting guide](https://docs.microsoft.com/azure/storage/common/storage-explorer-troubleshooting) and the [release notes](https://github.com/microsoft/AzureStorageExplorer/releases).
- [X] I have searched for [similar issues](https://github.com/Microsoft/AzureStorageExplorer/issues).

### Storage Explorer Version

1.34.0

### Architecture

x64

### Platform

Windows

### OS Version

Windows 11

### Bug Description

I am trying to use storage explorer to download some files from a blob. I have disable my OS proxy settings, i have disable proxy settings via registry key as well, Under the settings of storage explorer i have it setup to do not use proxy. Also under settings -> proxy -> i have it with do not use proxy.  Yet the connection fails due a failed connection "proxyconnect tcp"

### Steps to Reproduce

1. Launch explorer
2. Sign in to account
3. Disable proxy settings to use do not use proxy
4. Under settings proxy setup to not use proxy
5. OS proxy settings turned off
6. registry settings proxy settings set to 0

### Actual Experience

I am unable to download anything because the connection fails due to trying to use the proxy settings.

### Expected Experience

I want storage explorer to not use proxy settings.

**Comment by craxal** [comment]
Does the problem persist if you close and relaunch Storage Explorer after changing to "Do not use proxy"?

Does the problem persist if you change it to "Use system proxy" and your OS proxy is disabled?

Can you share your app logs here?

**Comment by purpler3m** [comment]
Yes, I tried closing and relaunching after changing do not use proxy and the problem persist.  
I just tried using "use system proxy" and my OS proxy disable but when i check the logs .azcopy logs i can see it is still attempting to go through proxy.

**Comment by MRayermannMSFT** [comment]
@purpler3m unfortunately there is a disconnect between how Storage Explorer handles proxies and how AzCopy handles proxies due to the two tools being implemented using different technologies. If you have successfully disabled Storage Explorer from using proxy, but you still see the AzCopy operations using proxy, then look for and check the "Disable AzCopy Proxy Usage" setting:

![Image](https://github.com/user-attachments/assets/f3b5c4e0-2842-430d-8232-bbbf78d987ee)

**Comment by purpler3m** [comment]
You are correct. That fix my issues. Disabling the AZCopy from using proxy worked. I left Storage Explorer Proxy settings with auto manage with use system proxy, even though my system doesnt have any proxy.  Thank you.

**Comment by craxal** [comment]
Let's discuss whether it makes sense to merge this setting with the rest of Storage Explorer's proxy settings.

**Comment by craxal** [comment]
Investigate to find a viable solution, then revisit for further plans. Start this early.
